pandas/core/indexes/datetimelike.py

```diff
@@ -99,6 +99,16 @@ class TimelikeOps(object):
             frequency like 'S' (second) not 'ME' (month end). See
             :ref:`frequency aliases <timeseries.offset_aliases>` for
             a list of possible `freq` values.
+        ambiguous : 'infer', bool-ndarray, 'NaT', default 'raise'
+            - 'infer' will attempt to infer fall dst-transition hours based on
+              order
+            - bool-ndarray where True signifies a DST time, False designates
+              a non-DST time (note that this flag is only applicable for
```

Comment:
versionadded

can you test for .dt accessors as well?

Reply:
The .dt accessor tests are here: https://github.com/pandas-dev/pandas/blob/ef263112fa9e7f8ac8a3b965fab7fb111adace20/pandas/tests/series/test_datetime_values.py#L234